metrics.go

```diff
@@ -205,6 +214,13 @@ func StartReader(file io.ReadCloser, output io.Writer, errorWriter io.Writer) {
 	}()
 }
 
+func SetupWithGeoHash(path string, stdout io.Writer, stderr io.Writer, additionalLabels ...string) error {
+	isGeoHashing = true
+	additionalLabels = append(additionalLabels, geo_hash)
+	SetupModule(path, stdout, stderr, additionalLabels...)
```

Comment:
We should return the result of SetupModule so the error is propogated.

Reply:
Revised to do so.